DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+%% LyX 2.2.3 created this file.  For more info, see http://www.lyx.org/.
+%% Do not edit unless you really know what you are doing.
+\documentclass[DM,lsstdraft,toc,authoryear]{lsstdoc}
+\ifx\hypersetup\undefined
+  \AtBeginDocument{%
+    \hypersetup{unicode=true}
+  }
+\else
+  \hypersetup{unicode=true}
+\fi
+
+\makeatletter
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% LyX specific LaTeX commands.
+
+\title[jointcal]{jointcal: Simultaneous Astrometry \& Photometry for thousands of
+Exposures with Large CCD Mosaics}
+\author{John Parejko (University of Washington), Pierre Astier (LPNHE/IN2P3/CNRS
+Paris)}
+\setDocRef{DMTN-036}
+\date{\today}
```

Comment:
When you release and tag this, fix the date.

Reply:
What does "release" count as? I'll merge to master, but this is by no means "complete".

Reply:
Release probably means "when the JIRA ticket is done". You don't want a tweak to the build system next year to change the document date.